### PR TITLE
Basic files for ticket_desks

### PR DESCRIPTION
**Pull request by lukasz-ligizynski**

### CODE REVIEW

**Comment by wmaciejak** [comment]
Be consistent in whole application and user name `ticket_desk` instead of `ticketdesk`

**Comment by lukasz-ligizynski** [comment]
> Be consistent in whole application and user name `ticket_desk` instead of `ticketdesk`

Yeah I learned it hard way while creating reservations 😅

File: app/controllers/ticketdesks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class TicketdesksController < ActionController::Base
```
Comment (wmaciejak) [inline app/controllers/ticketdesks_controller.rb]: TicketDesks (also filename - should be ticket_desks)

File: app/controllers/ticketdesks_controller.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  def index
+    tiicketdesk = TicketDesks::Repository.new.find_all
```
Comment (wmaciejak) [inline app/controllers/ticketdesks_controller.rb]: ticket_desk

File: app/controllers/ticketdesks_controller.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    render json: TicketDesks::Representer.new(tiicketdesk).basic
```
Comment (wmaciejak) [inline app/controllers/ticketdesks_controller.rb]: ticket_desk

File: app/controllers/ticketdesks_controller.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+  end
+
+  def ticketdesk_params
+    params.require(:ticketdesk).permit(:status)
```
Comment (wmaciejak) [inline app/controllers/ticketdesks_controller.rb]: ticket_desk x2

File: config/routes.rb [hunk 5]
```diff
@@ -7,4 +7,5 @@
   get '/tickets', to: 'tickets#index'
   get '/reservations', to: 'reservations#index'
   get '/create_hall', to: 'halls#new'
+  get '/ticket_desks', to: 'ticketdesks#index'
```
Comment (wmaciejak) [inline config/routes.rb]: ticket_desk#index

**Pull request merged**